pyaedt/application/Analysis3D.py

```diff
@@ -504,6 +504,94 @@ def copy_solid_bodies_from(self, design, assignment=None, vacuum=True, pec=True,
         self.modeler.refresh_all_ids()
         return True
 
+    @pyaedt_function_handler(filename="input_file")
+    def import_3d_cad(
+        self,
+        input_file,
+        healing=False,
+        refresh_all_ids=True,
+        import_materials=False,
+        create_lightweigth_part=False,
+        group_by_assembly=False,
+        create_group=True,
+        separate_disjoints_lumped_object=False,
+        import_free_surfaces=False,
+        point_coicidence_tolerance=1e-6,
+        heal_stl=True,
+        reduce_stl=False,
+        reduce_percentage=0,
+        reduce_error=0,
+        merge_planar_faces=True,
+    ):
+        """Import a CAD model.
+
+        Parameters
+        ----------
+        input_file : str
+            Full path and name of the CAD file.
+        healing : bool, optional
+            Whether to perform healing. The default is ``False``, in which
+            case healing is not performed.
+        healing : int, optional
+            Whether to perform healing. The default is ``0``, in which
+            case healing is not performed.
```

Comment:
```suggestion
        healing : bool, optional
            Whether to perform healing. The default is ``False``.
```
Duplicate entry removed (as well as unnecessary text).